crates/ruff_python_formatter/src/comments/format.rs

```diff
@@ -371,16 +369,10 @@ impl Format<PyFormatContext<'_>> for FormatTrailingEndOfLineComment<'_> {
             0
         } else {
             // Start with 2 because of the two leading spaces.
-            let mut width = 2;
-
-            // SAFETY: The formatted file is <= 4GB, and each comment should as well.
-            #[allow(clippy::cast_possible_truncation)]
-            for c in normalized_comment.chars() {
-                width += match c {
-                    '\t' => f.options().tab_width().value(),
-                    c => c.width().unwrap_or(0) as u32,
-                }
-            }
+            let width = TextWidth::from_text(&normalized_comment, f.options().tab_width())
+                .width()
+                .expect("Expected comment not to contain any newlines")
+                + 2;
```

Comment:
```suggestion
            let width = 2 + TextWidth::from_text(&normalized_comment, f.options().tab_width())
                .width()
                .expect("Expected comment not to contain any newlines");
```
more intuitive for me because it's first the two spaces and then the comment. maybe even `"  ".text_len()`.